dallinger/utils.py

```diff
@@ -816,6 +822,14 @@ def map_locations_to(self, destination):
         raise NotImplementedError()
 
 
+def is_broken_symlink(path):
```

Comment:
Should we adopt specifying method signatures?

Reply:
What do you mean by method signature?

Reply:
I thought of something like:
`def is_broken_symlink(path: str) -> bool:`

Reply:
OK added